doc/contrib/pycharm/README.md

```diff
@@ -1,21 +1,204 @@
-# Setup development environment for VSCode
+# Setup development environment for PyCharm
+**Summary**: in this tutorial, you’ll learn how to install the `pipenv` packaging tool on Windows.
 
+# Install pipenv PyCharm
 
-## Devcontainer
-This project includes a [development container](https://code.visualstudio.com/docs/remote/containers) configuration, which lets you run the full-featured development environment inside a [Docker container](https://docker.com/). This is the fastest and easiest solution to get a development environment up and running.
+To install pipenv local to use in PyCharm, the following has to be done:
 
-### System Requirements
-- Docker running on local machine
-- VSCode Extenstion: [Remote Development](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack)
+- [ ] Install python for Windows
+- [ ] Install  Microsoft C++ Build Tools
+- [ ] Install pipenv for Windows
+- [ ] Configure a Pipenv environment
+- [ ] Setup Configuration in PyCharm
+ 
+---
+## Install python for Windows
+The Fenris project uses 3.8.10
 
-### Getting started
+Link:  https://www.python.org/downloads/release/python-3810/
 
-When system requirements are met, open the codeproject within VSCode. Upon opening the project, VSCode might prombt you that this project contains a devcontainer and if you wanna open this, press yes. If not open VSCode [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) using `Ctrl + Shift + P` or `F1` then enter:
+Remember to mark the "Add Python to enviroment variables"
 
-    > Remote-Containers: Rebuild Container
+---
+## Install  Microsoft C++ Build Tools
 
+Microsoft C++ Build Tools is required to run the serpyco package in PyCharm
 
-You should then have a full-featured development environment up and running.
+Link to Tools for Visual Studio 2022
 
-### Git tips
-In some cases there might be problems, when pushing commits to remote git repository. These might be addessed [here](https://code.visualstudio.com/docs/remote/containers#_working-with-git).
+https://visualstudio.microsoft.com/downloads/
+
+or
+
+Direct link;
+
+https://aka.ms/vs/17/release/vs_BuildTools.exe
+
+In the installation, Windows 10 SDK is required, however, problems can occur if the others default are not choosen.  
+
+---
+## Install pipenv for Windows
+Link to the metode;
+https://www.pythontutorial.net/python-basics/install-pipenv-windows/
+
+### Prerequisites
```

Comment:
Move the `python -V` to the "Install python for Windows step" and reference that section.